src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -110,7 +109,8 @@ async function vaFacilityNext(state, dispatch) {
     return 'requestDateTime';
   }
 
-  throw new Error('Veteran not eligible for direct scheduling or requests');
```

Comment:
This situation is handled in the normal facilities page by redirecting to the facilities page to show the modal, so we no longer need this error message.